Gaijinizer patch/Map050.txt

```diff
@@ -493,8 +493,8 @@ previous girl.#MTLed
 > CONTEXT: Map050/events/46/pages/1/224/Dialogue/yuuma
 > CONTEXT: Map188/events/46/pages/0/291/Dialogue/yuuma
 > CONTEXT: Map188/events/46/pages/1/224/Dialogue/yuuma
-But it's a bother..."Concert venue park" is
-just our direction.#MTLed
+But that's a pain... That "Concert venue park" is
```

Comment:
This should help it flow better...
>**_That's a pain... 
The park she's holding her "concert" in
is right along the path you need to go._**